index.d.ts

```diff
@@ -6,326 +6,360 @@
+  save(path: string, options?: SaveOptions): Promise<void>;
+  toDataURL(type?: string, options?: SaveOptions): string;
+  toBase64(type?: string, options?: SaveOptions): string;
+  toBlob(type?: string, quality?: number): Blob;
+  getCanvas(): HTMLCanvasElement;
+
+  checkProcessable(processName: string, options: object): void;
+  getRGBAData(options?: { clamped?: boolean }): Uint8Array | Uint8ClampedArray;
+
+  // extend
+  invert(options?: OutOrInplace): Image;
+  abs(options?: OutOrInplace): Image;
+  level(options?: {
+    channels?: SelectedChannels;
+    min?: number;
+    max?: number;
+  }): this;
+  // add
+  // subtract
+  // subtractImage
+  // multiply
+  // divide
+  // hypotenuse
+  // background
+  // flipX
+  // flipY
+
+  blurFilter(options?: { radius?: number }): Image;
+  medianFilter(options?: {
+    radius?: number;
+    border?: BorderHandling;
+    channels?: SelectedChannels;
+  }): Image;
+  gaussianFilter(options?: GaussianFilterOptions): Image;
+  gradientFilter(options?: GradientFilterOptions): Image;
+  sobelFilter(options?: GradientOptions): Image;
+  scharrFilter(options?: GradientOptions): Image;
+
+  dilate(options?: MorphologicalOptions): Image;
+  erode(options?: MorphologicalOptions): Image;
+  open(options?: MorphologicalOptions): Image;
+  close(options?: MorphologicalOptions): Image;
+  topHat(options?: MorphologicalOptions): Image;
+  blackHat(options?: MorphologicalOptions): Image;
+  morphologicalGradient(options?: MorphologicalOptions): Image;
+
+  // warpingFourPoints
+  crop(options?: CropOptions): Image;
+  // cropAlpha
+  resize(options?: ResizeOptions): Image;
+  // hsv
+  // hsl
+  // cmyk
+  // rgba8
+  grey(options?: GreyOptions): Image;
+  mask(options?: MaskOptions): Image;
+  // pad
+  // colorDepth
+  // setBorder
+  rotate(angle: number, options?: RotateOptions): Image;
+  rotateLeft(): Image;
+  rotateRight(): Image;
+
+  // getRow
+  // getColumn
+  // getMatrix
+  // setMatrix
+  // getPixelsArray
+  // getIntersection
+  // getClosestCommonParent
+  // getThreshold
+
+  // split
+  // getChannel
+  // combineChannels
+  // setChannel
+  // getSimilarity
+  // getPixelsGrid
+  // getBestMatch
+
+  // cannyEdge
+  convolution(kernel: Kernel, options?: ConvolutionOptions): Image;
+  // extract
+  // floodFill
+  // paintLabels
+  // paintMasks
+  // paintPoints
+  // paintPolyline
+  // paintPolylines
+  // paintPolygon
+  // paintPolygons
+
+  // countAlphaPixels
+  // monotoneChainConvexHull
+  // minimalBoundingRectangle
+  // getHistogram
+  // getHistograms
+  // getColorHistogram
+  // getMin
+  // getMax
+  // getSum
+  // getMoment
+  // getLocalMaxima
+  // getMedian
+  // getMean
+  // getPoints
+  // getRelativePosition
+}
+
+export declare class Stack extends Array<Image> {
+  static load: (files: string[]) => Stack;
+
+  averageImage: Stack;
+  // histogram
+  // histograms
+  // matchAndCrop
+  // max
+  maxImage: Image;
+  // median
+  // min
+  minImage: Image;
 }
```

Comment:
```suggestion
export declare class Stack extends Array<Image> {
  static load: (files: string[]) => Stack;

  constructor();
  constructor(images: Image[]);

  getAverageImage(): Stack;
  // histogram
  // histograms
  // matchAndCrop
  // max
  getMaxImage(): Image;
  // median
  // min
  getMinImage(): Image;
}
```

I just checked, there's no getter on Stacks